contracts/tokenomics/guages/LiquidityGaugeV5.vy

```diff
@@ -676,6 +676,7 @@ def add_reward(_reward_token: address, _distributor: address):
 
     self.reward_data[_reward_token].distributor = _distributor
     self.reward_tokens[reward_count] = _reward_token
+    self.reward_data[_reward_token].token = _reward_token
```

Comment:
I would also add `assert _reward_token != ZERO_ADDRESS` at top of this function to check for zero inputs

Reply:
Addressed in https://github.com/saddle-finance/saddle-contract/pull/579/commits/ad2495dcb322a0583b0707e2f45a8874d1a6261a